### PR TITLE
Update tags.js

### PR DESCRIPTION
**Pull request by crock**

### CODE REVIEW

File: config/tags.js [hunk 1]
```diff
@@ -6,6 +6,14 @@
 import { blue, green, orange, purple, red, yellow } from '../src/utils/colours'
 
 export const tags = {
+  oss: {
+    title: 'First Time Contributors',
+    color: blue,
+    description: `Contributing to reputable open-source projects is a great way to build experience in the tech industry without doing free work for clients.\n\n
+https://yourfirstpr.github.io/
+https://www.firsttimersonly.com/
+`
```
Comment (Banou26) [inline config/tags.js]: Should probably use multiple lines outside of the template literal tag for this.
Look at the other entries for the style guideline.

**Review by Banou26 — CHANGES_REQUESTED** [review]

File: config/tags.js (hunk 1 above)
Comment (saghen) [reply config/tags.js]: In my case, I [used an array and joined](https://github.com/redxtech/devmod/blob/master/config/tags.js#L25-L42)

Comment (redxtech) [reply config/tags.js]: Switch it to either of those (I'd probably go for the joined array) and I'll merge.

Comment (crock) [reply config/tags.js]: Done.

**Pull request merged**